File: 240320/코드네임/code-name.js
class User {
    constructor(code_name, score) {
        this.code_name = code_name;
        this.score = score;
    }
}

const fs = require("fs");
const input = fs.readFileSync(0).toString().trim().split('\n');

// 변수 선언 및 입력
const users = [];

for (let i = 0; i < 5; i++) {
    const [code_name, score] = input[i].split(' ');
    users.push(new User(code_name, parseInt(score)));
}


// 최소 점수를 갖는 유저 찾기
let min_idx = 0;
for (let i = 1; i < 5; i++) {
    if (users[min_idx].score > users[i].score) {
        min_idx = i;
    }
}

// 출력
console.log(users[min_idx].code_name, users[min_idx].score);

File: 240516/코드네임/code-name.py
class User:
    def __init__(self, name, score):
        self.name = name
        self.score = score

people = []

for _ in range(5):
    name, score = input().split()
    people.append( User(name, int(score)) )
    
people.sort(key=lambda x: x.score)

people1 = people[0]

print(people[0].name, people[0].score)